impress-org/givewp
# Campaigns UI is Scaffolded

Resolves #5271

## Description



- Adds a "Campaigns" domain to `src/`.
- Registers a "Campaign Editor" block.
- Registers a "Campaign Progress Bar" block.
- Removes previous "AdminView" implementation, which is no longer in use.
- Removes the `resources/js/app` directory, which is no longer in use.

Note: This iteration leverages existing blocks from `@wordpress/block-library` to spike customization and settings. The intent is that these components will be replaced as needed to support specific workflows and editing constraints.

## Affects



## Visuals



![image](https://user-images.githubusercontent.com/10858303/93905589-205fd980-fcc9-11ea-91e7-bff0e63094e7.png)

## Pre-review Checklist




## Testing Instructions



- Run `npm run dev`
- Go to "Donations" -> "Campaigns" submenu item.
- Click "Add New Campaign"
- See Campaign Card UI

## User Documentation
